src/awesome-texmacs.tm

```diff
@@ -1,11 +1,13 @@
 <TeXmacs|1.99.14>
 
-<style|notes>
+<style|browser>
 
 <\body>
-  <hlink|[index]|./index.tm><chapter*|Awesome GNU TeXmacs>
+  <hlink|[index]|./index.html><label|auto-1>
 
-  <section*|Plugins>
+  <chapter*|<label|auto-1>Awesome GNU TeXmacs>
+
+  <section*|<label|auto-2><label|auto-2>Plugins>
```

Comment:
If you do not make meaningful changes, then better to remove it from your commit. Like here?

Reply:
All of these changes happened without me realizing it. Perhaps it is better if I re-fork the repository and start from scratch, but I need to understand what happened, otherwise the program will keep behaving in the same way.